spec/rspec/expectations/extensions/kernel_spec.rb

```diff
@@ -1,79 +0,0 @@
-RSpec.describe Object, "#should" do
-  before(:example) do
-    @target = "target"
-    @matcher = double("matcher")
-    allow(@matcher).to receive(:matches?).and_return(true)
-    allow(@matcher).to receive(:failure_message)
-  end
-
-  it "accepts and interacts with a matcher" do
-    expect(@matcher).to receive(:matches?).with(@target).and_return(true)
-    expect(@target).to @matcher
-  end
```

Comment:
This is covered by more specific specs:
 - https://github.com/rspec/rspec-expectations/blob/bf9db76a293852a23ec4c46b55bf4ea58a66288c/spec/rspec/expectations/handler_spec.rb#L77
 - https://github.com/rspec/rspec-expectations/blob/bf9db76a293852a23ec4c46b55bf4ea58a66288c/spec/rspec/matchers/dsl_spec.rb#L825